mlflow/types/llm.py

```diff
@@ -395,8 +602,31 @@ def __post_init__(self):
                 Object(
                     [
                         Property("role", DataType.string),
-                        Property("content", DataType.string),
+                        Property("content", DataType.string, False),
```

Comment:
`content` is not required if `refusal` or `tool_calls` is present

Reply:
This sounds like another tricky case we need to consider in model signature :)